### PR TITLE
Add Ratvar

### PR DESCRIPTION
**Pull request by MACMAN2003**
## About the PR

Added Ratvar
## Why / Balance

Adding Ratvar will restore balance to the universe by filling the Ratvar shaped hole in it, and it's one less thing for whoever implements clockcult to do.
## Media

image
Nature is healing... balance has been restored.

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

he's effectively a nar'sie reskin, only found in the sandbox spawn menu, so i don't think this is worthy of a cl

### CODE REVIEW

File: Resources/Prototypes/Entities/Mobs/Player/ratvar.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+- type: entity
+  abstract: true
+  id: MobRatvarBase #can't have the clockwork justiciar be associated with his (im)mortal enemy
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Mobs/Player/ratvar.yml:3]: make a spawning animation too tg should have it

Comment (MACMAN2003) [reply Resources/Prototypes/Entities/Mobs/Player/ratvar.yml:3]: tg doesn't have a spawning animation, I'll make one.

Comment (keronshb) [reply Resources/Prototypes/Entities/Mobs/Player/ratvar.yml:3]: Citadel might have one

Comment (MACMAN2003) [reply Resources/Prototypes/Entities/Mobs/Player/ratvar.yml:3]: I checked the citdael github and they don't have a spawning animation for ratvar

File: Resources/Prototypes/Entities/Mobs/Player/ratvar.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    maxZoom: 2.0,2.0
+  - type: Fixtures
+    fixtures:
+      EventHorizonCollider:
+        shape:
+          !type:PhysShapeCircle
+            radius: 5
+        hard: false
+        restitution: 0.8
+        density: 99999
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Mobs/Player/ratvar.yml:36]: Why density so high when it's not even collidable.

**Comment by K-Dynamic** [comment]
Coggers gamemode when

**Comment by MACMAN2003** [comment]
i couldn't find a spawning animation in TG or Citadel's repositories so i had to make one

**Comment by deltanedas** [comment]
it looks sick :+1:

**Comment by metalgearsloth** [comment]
> i couldn't find a spawning animation in TG or Citadel's repositories so i had to make one

https://scrubby.melonmesa.com/icon/search/ratvar?codebase=citadel

**Comment by deltanedas** [comment]
cit one is a lot worse than the one made for this pr, its just le generic youtuber transition effect

**Comment by mirrorcult** [comment]
the one they made is cooler so idc

**Comment by QuietlyWhisper** [comment]
i like the one in this pr better

**Pull request merged**